src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,31 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            data-cy="Tab"
+            className={selectedTab.id === tab.id && 'is-active'}
```

Comment:
Use classnames library to set classes conditionally

Reply:
Good point, fix this
[CN](https://www.npmjs.com/package/classnames)